### PR TITLE
Adaptive CSS - Media Query

### PR DESCRIPTION
**Pull request by aperacaula**

### CODE REVIEW

**Comment by mikelpmc** [comment]
👍

**Review by mikelpmc — APPROVED** [review]

File: staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+*{
```
Comment (manuelbarzi) [inline staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css]: MOVE common css rules (mobile / desktop) to main.css

File: staff/aperacaula/css-layout-simple-adaptive/styles/mobile.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+*{
```
Comment (manuelbarzi) [inline staff/aperacaula/css-layout-simple-adaptive/styles/mobile.css]: MOVE common css rules (mobile / desktop) to main.css

File: staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+aside{
+    height: 100%;
+    width: 20%;
+    float: right;
+}
+
+footer{
+    height: 20%;
+    width: 100%;
```
Comment (manuelbarzi) [inline staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css]: TIP is ```width: 100%```required?

File: staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+section{
+    height: 100%;
+    width: 60%;
+    float: left;
+}
+
+aside{
+    height: 100%;
```
Comment (manuelbarzi) [inline staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css]: TIP same css rule in different selectors. why not grouping them?

SEE Grouping Selectors https://www.w3schools.com/css/css_syntax.asp

File: staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css (hunk 3 above)
Comment (aperacaula) [reply staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css]: 👌🏻not required

File: staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css (hunk 4 above)
Comment (aperacaula) [reply staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css]: ok, I see! nevertheless, if it's only the height and float property they share, it's quite the same to write it apart isn't it? or how would it be?:
nav, section{ 
    height 100%
    float left
}
a nd then the particular widht for each?

File: staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css (hunk 1 above)
Comment (aperacaula) [reply staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css]: done!

**Comment by manuelbarzi** [comment]
👍

**Pull request merged**